DigitalLearningSolutions.Data.Tests/Services/PasswordResetServiceTests.cs

```diff
@@ -14,7 +14,6 @@
 
     public class PasswordResetServiceTests
     {
-        private IConfigService configService;
```

Comment:
configService was left in after previous changes and was no longer being used, so I removed it here.